src/main/java/seedu/address/logic/commands/AddSellerProfile.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static java.util.Objects.requireNonNullElse;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Seller;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddSellerProfile extends Command {
```

Comment:
Could be possible to extend AddSellerProfile from AddClientProfile, prevents code reuse with AddBuyerProfile